frame/assets/src/impl_fungibles.rs

```diff
@@ -179,22 +179,6 @@ impl<T: Config<I>, I: 'static> fungibles::Create<T::AccountId> for Pallet<T, I>
 	}
 }
 
-impl<T: Config<I>, I: 'static> fungibles::Destroy<T::AccountId> for Pallet<T, I> {
-	type DestroyWitness = DestroyWitness;
-
-	fn get_destroy_witness(asset: &T::AssetId) -> Option<Self::DestroyWitness> {
-		Asset::<T, I>::get(asset).map(|asset_details| asset_details.destroy_witness())
-	}
-
-	fn destroy(
```

Comment:
Removal of that method would be a breaking change for the existing integrations, for those who use our pallet and for other similar assets pallets like orml. What could we do about this?

Reply:
Yeah, that's true.

Though my understanding from @joepetrowski is that this particular functionality isn't really used in the wild yet. 
What do you propose we do about it?

Reply:
Maybe we could keep that destroy method for compatibility, but change the implementation and allow to call it if there are less than T::RemoveItemsLimit items to remove? In that case, this method would consist of the items amount check and will simply call `do_start_destroy(), do_destroy_accounts(), do_destroy_approvals(), do_finish_destroy()`.

Reply:
Yeah, that makes sense. Though it's always to be a bit annoying maintaining 2 ways of doing the same thing.

Reply:
agree

Reply:
I weighed it in with @joepetrowski and since it seems no one is using the destroy extrinsic yet, it should be fine to break it.

Reply:
https://statemint.subscan.io/event?address=&module=assets&event=destroyed&startDate=&endDate=&startBlock=&endBlock=&timeType=date&version=9290